Framework/API/src/FileProperty.cpp

```diff
@@ -216,7 +214,10 @@ bool FileProperty::isOptional() const {
  * An empty string indicates success.
  */
 std::string FileProperty::setValue(const std::string &propValue) {
-  std::string strippedValue = Kernel::Strings::strip(propValue);
```

Comment:
Could I just double check my understanding. I thought `strip` just called `boost::trim` that only trims leading and trailing whitespaces (an example on compiler explorer: https://gcc.godbolt.org/z/r3TdY8zoj) and not all whitespace?
I am assuming I have slightly misunderstood the input but I wanted to check to answer the question about `Load`.

Reply:
I had a look into this and you are correct. The changes proposed in this PR regarding `FileProperty` are actually not necessary for this fix to work, and the only part of the code that needs touching considers the string parser. I propose then to revert changes done to `FileProperty` and connections to it done in MultiFileProperty and leave in only bits that allow control over removing all whitespaces, which is defined in the `MultiFileNameParser`.

Reply:
Sounds good to me. Thanks for checking and clarifying here.